### PR TITLE
Contributing.wiki

### PR DESCRIPTION
**Pull request by leovalais**

### CODE REVIEW

File: src/contributing.wiki [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+The documentation format is the wikicréole. Please find the reference
+[[https://ocsigen.org/html_of_wiki/2.0/manual/intro|here]]. You can generate
+the documentation of your API  directly in Ocsigen's wikicréole using {{{ocamldoc}}}.
+Do not hesitate to ask for help if you experience any trouble!
+
+Remember, you have to issue **2** pull requests---one for your code on ##master##
+and one for its documentation on ##wikidoc##---for your contribution to be
+accepted!
+
+==For package maintainers
```
Comment (balat) [inline src/contributing.wiki]:
```suggestion
==Create a release (for package maintainers)
```

File: src/contributing.wiki [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+# Make sure the documentation on ##wikidoc## is **up to date** with the code on ##master##.
+# On ##wikidoc##, {{{cp dev X.Y.Z}}}, where {{{X.Y.Z}}} is the new version's number.
+# On ##master##, update opam configuration file.
+# {{{git tag}}}
+# Create a release on GitHub.
+# Publish with opam and wait for the package to be accepted.
```
Comment (balat) [inline src/contributing.wiki]:
```suggestion
# {{{opam publish}}} and wait for the package to be accepted.
```

File: src/contributing.wiki [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+the documentation of your API  directly in Ocsigen's wikicréole using {{{ocamldoc}}}.
+Do not hesitate to ask for help if you experience any trouble!
+
+Remember, you have to issue **2** pull requests---one for your code on ##master##
+and one for its documentation on ##wikidoc##---for your contribution to be
+accepted!
+
+==For package maintainers
+# Make sure the documentation on ##wikidoc## is **up to date** with the code on ##master##.
+# On ##wikidoc##, {{{cp dev X.Y.Z}}}, where {{{X.Y.Z}}} is the new version's number.
```
Comment (balat) [inline src/contributing.wiki]: cp -a ?

**Comment by leovalais** [comment]
@balat as the blog is no longer maintained, should I remove its link in the page footer?

**Pull request merged**